packages/core/src/shared/sam/debugger/pythonSamDebug.ts

```diff
@@ -154,20 +154,13 @@ function getPythonExeAndBootstrap(runtime: Runtime) {
     // unfortunately new 'Image'-base images did not standardize the paths
     // https://github.com/aws/aws-sam-cli/blob/7d5101a8edeb575b6925f9adecf28f47793c403c/samcli/local/docker/lambda_debug_settings.py
     switch (runtime) {
-        case 'python3.7':
```

Comment:
There are still tests for python3.7, so if we remove support here we should remove those too https://github.com/aws/aws-toolkit-vscode/blob/72d167b218bc9804a0a0a6771622b9fa40901eca/packages/core/src/test/shared/sam/debugger/samDebugConfigProvider.test.ts#L2093